### DISCUSSION TITLE
How to update to the latest webOS HyperHDR?

### DISCUSSION THREAD

**Discussion by NatyaSadella** [awawa-dev/HyperHDR#444] [Q&A]
Hi! 

Every time I open up the HyperHDR web configurator, I see a notification that I use an older HyperHDR Version. How to update to the latest HyperHDR Version when it‘s running on webOS? The latest version could solve a problem with dark scenes. 

Thank you

**Comment by awawa-dev** [comment]
Hi
Unfortunately I don't maintain any webos repo and I don't know either who could help you currently with preparing & installing newer version.
